From a5929130223973636f3fd25fbfaf2953f2ec96a9 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Fri, 25 Oct 2019 13:05:17 +0000
Subject: [PATCH] http: add missing header bootlegged by boost < 1.72

httpserver.cpp:74:10: error: no template named 'deque' in namespace 'std'
    std::deque<std::unique_ptr<WorkItem>> queue;
    ~~~~~^

[Retrieved from:
https://github.com/bitcoin/bitcoin/commit/a5929130223973636f3fd25fbfaf2953f2ec96a9]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 src/httpserver.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/httpserver.cpp b/src/httpserver.cpp
index 81137771871f..d9c711332358 100644
--- a/src/httpserver.cpp
+++ b/src/httpserver.cpp
@@ -15,6 +15,7 @@
 #include <sync.h>
 #include <ui_interface.h>
 
+#include <deque>
 #include <memory>
 #include <stdio.h>
 #include <stdlib.h>
